config/ftbquests/quests/chapters/late_game.snbt

```diff
@@ -1381,20 +1462,20 @@
 				type: "item"
 			}]
 			size: 1.5d
-			subtitle: "You only need complete one of this quest's tasks to move on"
+			subtitle: "You only need to complete one of this quest's tasks to move on"
```

Comment:
task ids are different because I attempted to achieve what the subtitle claimed (EITHER bucket of acid OR [Ke AND Nq+ AND *Nq*]) but failed

Reply:
Perhaps it would be best to simply require any of: 
- Enriched Naquadah Dust
- Enriched Naquadah ingot
- Trinium Dust
- Trinium ingot

If you're doing Naqline, you're getting one or more of the ingots as a product even before Trinium coils.
Even though you can get Naquadria dust at this point, it doesn't become relevant until a few quests later when those aforementioned Trinium coils are acquired so you can turn dust into ingot.